### DISCUSSION TITLE
data evaluation BIOMOD_modeling

### DISCUSSION THREAD

**Discussion by LindeNozomiLeo** [biomodhub/biomod2#280] [Q&A]
Hi,

I am trying to run some ensemble models and arrive at the BIOMOD_modeling output with a question.

I have set the do.full.models to FALSE, because I want to make sure my models get split and evaluated. However, when looking at the saved files on my hard drive I only see Full models for the different PA datasets and algorithms. It also gives only these files as computed models when looking at the output of the function.

How do I make sure the models actually get evaluated the way I want them to?

Additionally, I have seen examples online where CV.strategy, CV.nb.rep and CV.perc get added as arguments. Do these crossvalidation arguments provide additional output on the models?

Thanks in advance

**Comment by rpatin** [comment]
Hi @LindeNozomiLeo,
I opened an issue to answer your question (https://github.com/biomodhub/biomod2/issues/281)
Best,
Rémi
